x/programs/cmd/simulator/cmd/root.go

```diff
@@ -214,7 +214,7 @@ func (s *Simulator) Init() error {
 	loggingConfig.LogLevel = typedLogLevel
 	loggingConfig.Directory = path.Join(basePath, "logs-"+nodeID.String())
 	loggingConfig.LogFormat = logging.JSON
-	loggingConfig.DisableWriterDisplaying = !*s.enableWriterDisplaying
+	loggingConfig.DisableWriterDisplaying = *s.enableWriterDisplaying
```

Comment:
@iFrostizz, is this correct?

Reply:
Not really... I think that we should just enable it on the client
```rust
        let Child { stdin, stdout, .. } = Command::new(self.path)
            .arg("interpreter")
            .arg("--cleanup")
            .arg("--enable-stdout-logs")
            .arg("--log-level")
            .arg("error")
            .stdin(Stdio::piped())
            .stdout(Stdio::piped())
            .spawn()?;
```

Reply:
I just got rid of the change